packages/node_modules/@node-red/nodes/core/function/10-function.html

```diff
@@ -82,6 +82,11 @@
                 <input id="node-input-outputs" style="width: 60px;" value="1">
             </div>
 
+            <div class="form-row">
+                <label for="node-input-timeout"><i class="fa fa-clock"></i> <span data-i18n="function.label.timeout"></span></label>
+                <input id="node-input-timeout" style="width: 60px;" value="1000">
```

Comment:
```suggestion
                <input id="node-input-timeout" style="width: 60px;" data-i18n="[placeholder]join.seconds">
```

Use a placeholder to indicate the units. This is consistent with other core nodes, such as the join node.